content/working_with/official_plugins/Utilities/rollback.md

```diff
@@ -0,0 +1,340 @@
+---
+layout: bt_wiki
+title: Rollback Plugin
+category: Official Plugins
+draft: false
+weight: 100
+---
+{{% note %}}
+These features are part of the **utilities plugin**.
+{{% /note %}}
+
+# Cloudify Utilities: Rollback Workflow
+
+## Description
+Add support for rollback node instances that exists in [unresolved states](#rollback-workflow) due to failure in install workflow.
+Also, wrappers workflows for {{< param product_name >}} lifecycle operations introduced.
+
+## Prerequisites
+* Tested with Cloudify 5.1.1.
+
+{{% warning title="Deprecation Warning" %}}
+  On {{< param product_name >}} 5.2 (future version), Rollback Workflow will be deprecated in Utilities plugin and will become a built-in workflow. 
+{{% /warning %}}
+  
+## Supported workflows
+
+### Rollback workflow
+
+Rollback workflow will look at each node state, decide if the node state
+is unresolved, and for those that are, execute the corresponding node
+operation that will get us back to a resolved node state.
+
+Unresolved node instance states are:
+
+* creating.
+* configuring.
+* starting.
+
+
+After rollback, `creating` and `configuring` node instances become `uninitialized`.
+`starting` node instances become `uninitialized`. 
```

Comment:
Why break it down to two cases if they behave the same?

Reply:
My mistake!!Thx!